src/base/castletimeutils.pas

```diff
@@ -591,28 +591,28 @@ function Timer: TTimerResult;
 {$ifdef ANDROID}
 var
   { Note that using this makes Timer not thread-safe
-    (but we neved guaranteed in the interface that it's thread-safe...). }
+    (but we neved guaranteed in the interface that it's thread-safe...).
+    This variables can be removed once everyone confirms that the solution is
+    always working.
```

Comment:
Improve the comment "solution" -> "solution of using CLOCK_MONOTONIC", to remember in the future what solution does this mention :)